travis/inject_version.sh

```diff
@@ -4,7 +4,7 @@
 # No need to change anything below the replacer variable.
 # Just change the value of replacer, the script will do everything for you
  
```

Comment:
/// Dusk here - I don't see anything wrong so far.

Reply:
Actually I don't think WE should change that, we set the versioning via Git tags already, what's the point of this?

Reply:
The 'Send Feedback' button will not appear if the word "beta" doesn't appear in `config.version`.

Reply:
but we override this nonetheless, which doesn't make sense

Reply:
Either suffix all tags with `-beta` then.

Reply:
No need to worry. I just removed the check and made it a permanent button.